NotifExtSmpl01/ViewController.swift

```diff
@@ -12,7 +12,24 @@ class ViewController: UIViewController {
 
     override func viewDidLoad() {
         super.viewDidLoad()
-        // Do any additional setup after loading the view, typically from a nib.
+        
+        let foo: String?
+        foo = nil
+        
+        var b: String? = "nobita"
+        b = nil; b="doraemon"
+        
+        var param = 1
+        
+        if param == 1 {
+            param += 1
+            
+        } else if param == 2 {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either merge this branch with the identical one on line 24 or change one of the implementations. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=swift%3AS1871)